### PR TITLE
Test claim amount and total claim amount when making claim

### PR DESCRIPTION
**Pull request by pragnesh-patel**
Check so that we have values and not £0.00

### CODE REVIEW

File: src/tests/citizen/claim/steps/claim.ts [hunk 1]
```diff
@@ -63,9 +68,15 @@ module.exports = {
...
+  getClaimAmount () {
+    claimAmount = parseFloat(claimant.claimAmount.amount1) + parseFloat(claimant.claimAmount.amount2) + parseFloat(claimant.claimAmount.amount3)
+    claimAmount = claimAmount.toFixed(2)
+    return claimAmount
+  },
+
   getTotalClaimAmount () {
-    totalClaimAmount = parseFloat(claimant.claimAmount.amount1) + parseFloat(claimant.claimAmount.amount2) + parseFloat(claimant.claimAmount.amount3)
-    totalClaimAmount = totalClaimAmount + fee
+    totalClaimAmount = parseFloat(this.getClaimAmount()) + fee
```
Comment (yarpo) [inline src/tests/citizen/claim/steps/claim.ts]: why do you use `parseFloat` if it's returned from `getClaimAmount()` where you make sure it's float?

Comment (pragnesh-patel) [reply src/tests/citizen/claim/steps/claim.ts]: If I didnt it was crashing, as it couldnt do a toFixed(2) , so had to make it explicit that this was a numeric value

Comment (pragnesh-patel) [reply src/tests/citizen/claim/steps/claim.ts]: @yarpo totalClaimAmount = totalClaimAmount.toFixed(2) was crashing. It didnt think it was a numeric

Comment (pragnesh-patel) [reply src/tests/citizen/claim/steps/claim.ts]: @robertparkinson spotted this when I needed help

Comment (rlewan) [reply src/tests/citizen/claim/steps/claim.ts]: `toFixed` returns a `string`.

**Pull request merged**